diff --git a/src/utils.h b/src/utils.h
index 9a69c8e..1a4f58c 100644
--- a/src/utils.h
+++ b/src/utils.h
@@ -105,7 +105,7 @@ class Timer
          */
         void start()
         {
-            time = std::chrono::high_resolution_clock::now();
+            time = std::chrono::system_clock::now();
         }
 
         /*
@@ -118,7 +118,7 @@ class Timer
          */
         void stop(const std::string& text)
         {
-            auto end = std::chrono::high_resolution_clock::now();
+            auto end = std::chrono::system_clock::now();
             std::cout << "Time passed: "
                 << std::chrono::duration_cast<std::chrono::milliseconds>(
                        end - time
